src/sentry/static/sentry/app/components/search/index.jsx

```diff
@@ -92,6 +92,11 @@ class Search extends React.Component {
 
     if (!to) return;
 
+    if (to.startsWith('http')) {
```

Comment:
Do we have a polyfill for this? I don't think this exists in IE11.

Reply:
Yeah, it's polyfilled by `@babel/polyfill` which uses corejs iirc